datajunction-server/datajunction_server/database/history.py

```diff
@@ -35,14 +33,14 @@ class History(Base):
         BigInteger().with_variant(Integer, "sqlite"),
         primary_key=True,
     )
-    entity_type: Mapped[Optional[EntityType]] = mapped_column(
-        Enum(EntityType),
+    entity_type: Mapped[Optional[str]] = mapped_column(
+        String(20),
```

Comment:
Why only 20?

Reply:
No reason, I just thought that the postgres default of infinite length seemed unnecessary for what will just be a string representation of an enum value. Should I make this larger? Or maybe don't specify the length at all?